lib/pedicel/base.rb

```diff
@@ -89,22 +83,22 @@ def decrypt_aes(key:)
       else
         require 'aes256gcm_decrypt'
 
-        Aes256GcmDecrypt::decrypt(encrypted_data, key)
+        Aes256GcmDecrypt.decrypt(encrypted_data, key)
```

Comment:
Hmm, both `.` and `::` work for invoking a singleton method in a module. I wonder which one is the most blessed? :slightly_smiling_face:

Reply:
`.` is: https://github.com/bbatsov/ruby-style-guide#double-colons